fetch.bs

```diff
@@ -4585,6 +4620,9 @@ steps. They return a <a for=/>response</a>.
 
     <p>is true; otherwise false.
 
+   <li><p>If <a>Cross-Origin-Embedder-Policy allows credentials</a> with <var>request</var> is
```

Comment:
maybe "returns" is better than "is" here.

Reply:
Done.